docker/resource_docker_container_funcs.go

```diff
@@ -43,17 +43,15 @@ func resourceDockerContainerCreate(d *schema.ResourceData, meta interface{}) err
 		Image:      image,
 		Hostname:   d.Get("hostname").(string),
 		Domainname: d.Get("domainname").(string),
-		Tty:        d.Get("tty").(bool),
-		OpenStdin:  d.Get("stdin_open").(bool),
 	}
 
-	// if value, ok := d["tty"]; ok {
-	// 	config.Tty = value.(bool)
-	// }
+	if value, ok := d.GetOk("tty"); ok {
+		config.Tty = value.(bool)
+	}
 
-	// if value, ok := d["stdin_open"]; ok {
-	// 	config.OpenStdin = d.Get("stdin_open").(bool)
-	// }
+	if value, ok := d.GetOk("stdin_open"); ok {
+		config.OpenStdin = value.(bool)
+	}
```

Comment:
We can simply use `d.Get()` instead of `d.GetOk` in this case.
So this change is unneeded.

https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/schema#ResourceData.Get
> If the key does exist in the schema but doesn't exist in the configuration, then the default value for that type will be returned. For strings, this is "", for numbers it is 0, etc.